documentation/src/main/asciidoc/topics/ref_discovery_kube_ping.adoc

```diff
@@ -1,6 +1,7 @@
+[id='discovery_kubeping-{context}']
 = KUBE_PING
```

Comment:
KUBE_PING is old and we don't ship/productize it (DNS_PING is recommended). I would remove it from docs.